README.md

```diff
@@ -35,6 +45,28 @@
 - [ ] Release a production ready service
 - [ ] Release a beta of the style analyzer[s]
 
+### Code review study across GitHub
+
+- [ ] Conduct the study how people do code reviews on GitHub
+    - We need the numbers. E.g. what is the ratio of stupid style fixes compared to the overall? Clever fixes?
+
+### ML approach to source code style check
```

Comment:
This is key results of the unique ML goal, that is building the style analyzer, also I think this is to early to add this key results before we had the design meeting with the requirements.

Reply:
@eisokant You told me to add our OKRs and also asked my team members to add theirs. How on Earth it is possible if the whole list is rejected by CTO?

Reply:
Add key results, your OKRs to the tasks that belongs to your team